src/main/java/appeng/util/item/AEItemStack.java

```diff
@@ -146,7 +100,7 @@ public static AEItemStack create( final ItemStack stack )
 			return null;
 		}
 
-		return new AEItemStack( stack );
+		return new AEItemStack( AEItemStackRegistry.getRegisteredStack( stack ), stack.getCount() );
 	}
 
 	public static IAEItemStack loadItemStackFromPacket( final ByteBuf data ) throws IOException
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)